Rakefile

```diff
@@ -4,3 +4,58 @@ require "rspec/core/rake_task"
 RSpec::Core::RakeTask.new(:spec)
 
 task :default => :spec
+
+namespace :openapi do
+  IBM_CLOUD_GEMS = {
+    "ibm_cloud_iam"                 => "iam-identity-token-api.json",
+    "ibm_cloud_power"               => "power-cloud.json",
+    "ibm_cloud_resource_controller" => "resource-controller/resource-controller.json",
+    "ibm_cloud_vpc"                 => "vpc.json"
+  }
+
+  task :download_cli, [:version] do |_t, args|
+    version = args[:version] || "5.0.0-beta2"
+
+    target_file    = "openapi-generator-cli-#{version}"
+    target_symlink = "openapi-generator-cli"
+
+    next if File.exist?(target_file)
```

Comment:
`next` reads weird, but I understand why you have to do it... is it possible to use `break`?

Reply:
You still might want to apply the symlink anyway, even if we don't download the file (for example, in case someone tries to run this with an _old_ version that is already downloaded but the symlink is pointing to a newer version).  A symlink is inexpensive, so may be worth it to just do it all the time.

Reply:
Yeah I tried using `break` first that that fails for the same reason as `return`

Reply:
Okay switched this to `unless File.exist ` then always update the symlink